plugin/src/test/java/com/google/errorprone/xplat/checker/XplatBansTest.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
```

Comment:
Any reason we're using json instead of protos?

Reply:
The old Xplat checker used textproto files, and there is limited documentation on them, so Lukhnos suggested that I use json.